modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
Great, just [added](https://github.com/opencv/opencv_extra/pull/348), please kindly check and merge. I'll be waiting for your merge to complete the dnn wrappers pull request.